README.md

```diff
@@ -37,3 +37,24 @@ npm install -g webpack
 npm install
 ```
 
+## API routes
+
+## restaurant-specific routes
+GET /restaurant:id - loads restaurant data using unique restaurant name as parameter
+POST /restaurant - adds new restaurantdata to database, should include data like
+  restaurant name, reservation hours, number of tables, max reservation duration, 
+  max/min number of people for reservation, DOES IT EVEN HAVE A WAITLIST, allowed
+  to make more than one reservation per day
+PUT /restaurant:id - updates restaurant data according to unique restaurant id
+DELETE /restaurant:id - deletes restaurant data according to unique restaurant id
+
+## reservation specific routes
+GET /reservation/:id - get one reservation from the database, requires reservation id
```

Comment:
Would it make sense to also consider restaurant name as a parameter as well?

Reply:
I think it depends if I am trying to put all reservations from all restaurants into one table, or separate reservation tables among different restaurants. The latter seems more efficient, but then again that would be a lot of tables to make if each restaurant had a reservations table.